### ISSUE TITLE
Function key mappings don't work with Bluetooth keyboard

### ISSUE THREAD

**Issue by 8va** [pqrs-org/Karabiner-Elements#3008]
**TL/DR:**
In the "Function keys" tab of KE, my external (third-party) keyboard appears twice: one is its Bluetooth mode and one is its USB mode. Custom function key mappings don't work on the Bluetooth mode device, but do work on the USB one.

**Details:**
I'm trying to manually map f1–f12 to f1–f12 on my external bluetooth keyboard, and map f1–f12 to media keys on my internal MacBook Pro keyboard.

Karabiner's "Use all F1, F2, etc. keys as standard function keys" checkbox overrides custom mappings (like the media key mappings for the internal keyboard), so I have that unchecked.

Does anyone know a way to fix this?
